daemon/web/fonts/Rubik/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2015 The Rubik Project Authors,

```

Comment:
Since you'll be importing the font from online, you should delete the `/Rubik` directory